doc/manual/management_system.rst

```diff
@@ -54,6 +54,11 @@ Experiments are divided into three phases that are programmed by the user:
 
 The three phases of several experiments are then executed in a pipelined manner by the scheduler in the ARTIQ master: experiment A executes its preparation stage, then experiment A executes its running stage while experiment B executes its preparation stage, and so on.
 
+.. note:: 
+    The next experiment (B) may start :meth:`~artiq.language.environment.Experiment.run` ing before all events placed into FIFOs by the previous experiment (A) have triggered. These events can then trigger when experiment B is :meth:`~artiq.language.environment.Experiment.run` ing. Using :meth:`~artiq.coredevice.core.Core.reset` clears the FIFOs, discarding untriggered events, including those left over from A.
```

Comment:
They aren't FIFOs, just say "RTIO buffers".

Add something saying that this note is specific to experiments using the core device. Otherwise looks good to me.